Documentation/BuildInstructionsLadybird.md

```diff
@@ -143,8 +144,14 @@ WSL1 is known to have issues. If you run into problems, please use WSL2.
 
 MinGW/MSYS2 are not supported.
 
-Native Windows builds are not supported.
+# Native (Visual Studio Only)
+ you need choclatey package manager then run the commands all in an admin terminal of powershell
```

Comment:
given that the build in this state is only able to configure and not build, there needs to be a lot more information here about the non-working-ness.

Reply:
I did change this but not sure if it was to what you asked

Reply:
This is pretty good yeah. The sentences in the documentation should be a bit more grammatical though. Start with a capital, end with a period, etc. And make sure that we are spelling chocolatey consistently in filenames and comments and such.

Reply:
yea sorry about that Dyslexia can be catch me out sometime I'll get it checked out